Sources/MapboxCoreNavigation/Feedback/NavigationEventsManager.swift

```diff
@@ -380,15 +402,13 @@ open class NavigationEventsManager {
             sessionState.departureTimestamp = dataSource.rawLocation?.timestamp ?? Date()
         }
 
-        guard let attributes = (try? passiveNavigationEvent(type: .start)?.asDictionary()) as [String: Any]?? else { return }
-        mobileEventsManager.enqueueEvent(withName: NavigationEventTypeFreeDrive, attributes: attributes ?? [:])
-        mobileEventsManager.flush()
+        guard let attributes = try? passiveNavigationEvent(type: .start)?.asDictionary() else { return }
+        eventsAPI.sendImmediateEvent(with: attributes)
     }
 
     func sendPassiveNavigationStop() {
-        guard let attributes = (try? passiveNavigationEvent(type: .stop)?.asDictionary()) as [String: Any]?? else { return }
-        mobileEventsManager.enqueueEvent(withName: NavigationEventTypeFreeDrive, attributes: attributes ?? [:])
-        mobileEventsManager.flush()
+        guard let attributes = try? passiveNavigationEvent(type: .stop)?.asDictionary() else { return }
+        eventsAPI.sendImmediateEvent(with: attributes)
```

Comment:
Good question. I [asked](https://mapbox.slack.com/archives/C220EHL3W/p1657370299851729) the historical context, but seems like nobody knows. 
The enqueuing and flushing immediately produced the same effect, so we're not changing the behavior here.

Reply:
We’ve been flushing events this way ever since we added navigation events in #304, but I don’t recall any discussion about it. It might’ve just been an idea that we should try as hard as possible to send these events. 💪 The reroute and faster route events can happen many times in short succession, so I’m not surprised that we made them less immediate.